internal/service/accesslistapikey/resource_access_list_api_key_migration_test.go

```diff
@@ -31,21 +32,14 @@ func TestAccMigrationProjectAccesslistAPIKey_SettingIPAddress(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "ip_address", ipAddress),
 				),
 			},
-			{
-				ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
-				Config:                   configWithIPAddress(orgID, description, ipAddress),
-				ConfigPlanChecks: resource.ConfigPlanChecks{
-					PreApply: []plancheck.PlanCheck{
-						acc.DebugPlan(),
-						plancheck.ExpectEmptyPlan(),
-					},
-				},
-			},
+			mig.TestStepCheckEmptyPlan(configWithIPAddress(orgID, description, ipAddress)),
 		},
 	})
 }
 
 func TestAccMigrationProjectAccesslistAPIKey_SettingCIDRBlock(t *testing.T) {
+	mig.SkipIfVersionBelow(t, "1.14.0")
```

Comment:
Could you document this line? Might be beneficial to quickly know the why in the future

Reply:
+1, as an example we're also documenting SkipTestForCI so we know why they're being skipped

Reply:
added cooment